apps/studio/prisma/generated/types.ts

```diff
@@ -0,0 +1,52 @@
+import type { ColumnType } from "kysely"
```

Comment:
nit: use ts-morph, and generate a heading that states not to edit this file direcly as this is auto-generated. 

similar to what kr did (here)[https://github.com/opengovsg/activesg/pull/892/files]

Reply:
I think the point of `ts-morph` was to generate typings automatically cos they were using postgis which wasn't supported.

We can still do it, but given that this type is already automatically generated (and lives in a `generated` folder), there might not be much incremental value

Reply:
yea, skipping this for now